doc/source/ray-core/tasks.rst

```diff
@@ -78,25 +78,14 @@ Ray enables arbitrary functions to be executed asynchronously on separate Python
 
 .. _ray-object-refs:
 
-Passing object refs to remote functions
+Passing object refs to Ray tasks 
 ---------------------------------------
 
-**Object refs** can also be passed into remote functions. When the function actually gets executed, **the argument will be passed as the underlying Python value**. For example, take this function:
+In addition to values, `Object refs <objects.html>`__ can also be passed into remote functions. When the task gets executed, inside the function body **the argument will be the underlying value**. For example, take this function:
```

Comment:
I tried to clarify this and removed the _dereferencing_ terminology -- while this terminology is very clear to users of system programming languages, Python users typically don't know about pointers and/or dereferencing :)

Let me know if you think this is clear :)